hail/build.mill

```diff
@@ -107,7 +107,7 @@ trait HailScalaModule extends SbtModule with ScalafmtModule with ScalafixModule
   override def bspCompileClasspath: T[Agg[UnresolvedPath]] =
     super.bspCompileClasspath() ++ resources().map(p => UnresolvedPath.ResolvedPath(p.path))
 
-  trait HailTests extends SbtTests with TestNg with ScalafmtModule with ScalafixModule {
```

Comment:
I think you need to replace `SbtTests` with it's parent `ScalaTests`, which sets the scala version.

Reply:
What do you mean "sets up the scala version"?